### PR TITLE
eventDate clarified

### PR DESCRIPTION
**Pull request by cloveras**

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -2133,7 +2134,8 @@ components:
           example: '2019-06-01T00:00:00Z'
           description: |-
             The date and time the campaign ends.
-            Needs to be UTC.
+            This is a required field for some campaign types: `PERIOD_CAMPAIGN`, `EVENT_CAMPAIGN` and `ULL_FLEX_CAMPAIGN`.
```
Comment (skasti) [inline docs/openapi.yaml:2137]: This is not correct 😅
eventDate is currently only used by `EVENT_CAMPAIGN`, and is illegal for other types

File: docs/openapi.yaml [hunk 2]
```diff
@@ -2086,7 +2086,8 @@ components:
...
```
Comment (skasti) [inline docs/openapi.yaml:2089]: This is not correct 😅
eventDate is currently only used by `EVENT_CAMPAIGN`, and is illegal for other types

Comment (skasti) [reply docs/openapi.yaml:2089]: but if you add this type of info, you also need to add that `interval` and `end` is used by FULL_FLEX_CAMPAIGN and that `period`is used by PERIOD_CAMPAIGN, etc

Comment (skasti) [reply docs/openapi.yaml:2089]: Does openApi support inheritance? maybe we can create separate DTOs for the different campaign types and make that work somehow? 🤔

File: docs/openapi.yaml (hunk 1 above)
Comment (cloveras) [reply docs/openapi.yaml]: Nice. Fixed.

File: docs/openapi.yaml (hunk 2 above)
Comment (cloveras) [reply docs/openapi.yaml:2089]: Can you suggest the correction so I can commit it? Use the +/- functionality in the comment field.

Comment (cloveras) [reply docs/openapi.yaml:2089]: Just try to explain it in text for now, we can see what's possible with OpenAPI later.

Comment (cloveras) [reply docs/openapi.yaml]: OK?
```
          description: |-
            The date and time the campaign ends.
            This is a required field for `EVENT_CAMPAIGN`, and is illegal for other types.
            But: If you add this type of info, you also need to add that interval
            and end is used by `FULL_FLEX_CAMPAIGN` and that periodis used by `PERIOD_CAMPAIGN`, etc.
            Musst be UTC.
            ```

Comment (rognlien) [reply docs/openapi.yaml]: Just missing a space in: **that periodis used by**

File: docs/openapi.yaml [hunk 3]
```diff
@@ -2086,7 +2086,10 @@ components:
           example: '2019-06-01T00:00:00Z'
           description: |-
             The date and time the campaign ends.
-            Needs to be UTC.
+            This is a required field for `EVENT_CAMPAIGN`, and is illegal for other types.
+            But: If you add this type of info, you also need to add that interval
+            and end is used by `FULL_FLEX_CAMPAIGN` and that periodis used by `PERIOD_CAMPAIGN`, etc.
```
Comment (cloveras) [inline docs/openapi.yaml:2091]:
```suggestion
            and end is used by `FULL_FLEX_CAMPAIGN` and that period is used by `PERIOD_CAMPAIGN`, etc.
```

**Pull request closed without merging**